tools/languagelocalizer/README.md

```diff
@@ -0,0 +1,19 @@
+# Language string mapping tool
+
+This tool retrieves the native language and the localized language names for
+all the available translations.
```

Comment:
```suggestion
all the available locales.
```

A few nits in this document:
* language -> locale
* wiki data, wikidata, WikiData -> Wikidata